services/logs2notifications/internal/handler/useraction_events.go

```diff
@@ -0,0 +1,288 @@
+		}
+		return nil
+	} else {
+		if h.EnableDebug {
+			log.Printf("Sending email for '%v' event, user:%s\n", notification.Meta.Event, payload.Meta.Payload.UserActionEmailDetails.Email)
+		}
+	}
+
+	useractionEmailDetails := *payload.Meta.Payload.UserActionEmailDetails
+
+	switch notification.Meta.Event {
+	case "api:addUserToOrganization", "api:addAdminToOrganization":
+		return h.addAdminToOrganization(useractionEmailDetails)
+	case "api:removeUserFromOrganization", "api:removeAdminFromOrganization":
+		return h.removeAdminFromOrganization(useractionEmailDetails)
+	case "api:addPlatformRoleToUser":
+		return h.addPlatformRoleToUser(useractionEmailDetails)
+	case "api:removePlatformRoleFromUser":
+		return h.removePlatformRoleFromUser(useractionEmailDetails)
+	case "api:deleteSshKey", "api:deleteSshKeyById":
+		return h.addEditRemoveSshKeyEmailMessage(useractionEmailDetails, notification.Meta.Event, "deleted")
+	case "api:updateSshKey":
+		return h.addEditRemoveSshKeyEmailMessage(useractionEmailDetails, notification.Meta.Event, "updated")
+	case "api:addSshKey":
+		return h.addEditRemoveSshKeyEmailMessage(useractionEmailDetails, notification.Meta.Event, "added")
+	case "api:removeUserFromGroup", "api:addUserToGroup":
+		return h.groupAddRemoveEmailMessage(useractionEmailDetails, notification.Meta.Event)
+	}
+	return errors.New(fmt.Sprintf("Unable to match incoming notification: %v", notification))
+}
+
+func (h *Messaging) groupAddRemoveEmailMessage(valuesStruct UseractionEmailDetails, event string) error {
+
+	content := `
+<p>Hello,</p>
+
+<p>
+  You (<strong>{{.Name}}</strong>) have been {{.Action}} the group "{{.Groupname}}" in your Lagoon account.
+</p>
+
+<p>
+  If you have any questions or need assistance, please contact your organization manager.
+</p>
+`
+	action := "added to"
+	if event == "api:removeUserFromGroup" {
+		action = "removed from"
+	}
+
+	mainHTML, err := templateGenerator(content, struct {
+		Name      string
+		Action    string
+		Groupname string
+	}{
+		Name:      valuesStruct.Name,
+		Action:    action,
+		Groupname: valuesStruct.Groupname,
+	}, h.EmailBase64Logo)
+
+	if err != nil {
+		return err
+	}
+	plainText := fmt.Sprintf("%v %v group %v", valuesStruct.Name, action, valuesStruct.Groupname)
+	subject := fmt.Sprintf("User %s has been %v a group", valuesStruct.Name, action)
+	if err != nil {
+		return fmt.Errorf("error generating email template for %s event %s: %v", valuesStruct.Email, event, err)
+	}
+
+	err = h.simpleMail(valuesStruct.Email, subject, mainHTML, plainText)
+	if err != nil {
+		return fmt.Errorf("error sending email for %s event %s: %v", valuesStruct.Email, event, err)
+	}
+	return nil
+}
+
+func (h *Messaging) addEditRemoveSshKeyEmailMessage(valuesStruct UseractionEmailDetails, event, action string) error {
+
+	content := `
+<p>Hello,</p>
+
+<p>
+  You (<strong>{{.Name}}</strong>) have {{.Action}} an SSH key {{if .Keyname}} ({{.Keyname}}) {{end}} in your Lagoon account.
```

Comment:
Styling to match other messages

```suggestion
  You (<strong>{{.Name}}</strong>) have {{.Action}} an SSH key {{if .Keyname}} <strong>{{.Keyname}}</strong> {{end}}
```